src/components/Ribbon/Ribbon.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import Ribbon from './Ribbon';
+
+const meta = {
+  title: 'Components/Ribbon',
+  component: Ribbon,
+  argTypes: {
+    isConstrained: {
+      control: { type: 'boolean' }
+    },
+    variant: {
+      options: [null, 'informational' /* , 'carbon' */],
+      control: { type: 'radio' }
+    },
+    withShadow: {
+      control: { type: 'boolean' }
+    },
+    textAlign: {
+      options: ['left', 'center'],
+      control: { type: 'radio' }
+    }
+  },
```

Comment:
We might be able to remove these and let autodocs pick up this information automatically.

Reply:
You're right. It's exactly the same if removed.